visual_marker_mapping/CMakeLists.txt

```diff
@@ -37,13 +37,15 @@ set_property(TARGET visual_marker_mapping_lib PROPERTY CXX_STANDARD_REQUIRED ON)
 target_include_directories(visual_marker_mapping_lib PUBLIC
                            include
                            ${Boost_INCLUDE_DIRS}
-                           ${CERES_INCLUDE_DIRS})
+                           ${CERES_INCLUDE_DIRS}
+                           ${CMAKE_CURRENT_SOURCE_DIR}/umich_apriltags/apriltag-2016-10-21/)
```

Comment:
Die Verzeichnisstruktur in dem umich_apriltags ist etwas komisch. Du hast auch 2x include_directories dafuer: 1x in der Haupt CMakeLists.txt und einmal hier. Ich denke es waere besser das hier zu entfernen und nur in der Haupt-Datei zu haben. Ausserdem die Ordnerstruktur in umich_apriltags so umbauen, dass es dieses Unterverzeichnis apriltag-2016-10-21 nicht mehr gibt.